lucene/CHANGES.txt

```diff
@@ -247,6 +247,13 @@ Other
 * LUCENE-9627: Remove unused Lucene50FieldInfosFormat codec and small refactor some codecs
   to separate reading header/footer from reading content of the file. (Ignacio Vera)
 
+======================= Lucene 8.9.0 =======================
+
+New Features
+---------------------
+
+* LUCENE-9694: New tool for creating a deterministic index (Haoyu Zhai)
```

Comment:
Thanks for adding `CHANGES` entry!

Could we beef it up a bit, e.g.:

```
LUCENE-9694: New tool for creating a deterministic index to enabling benchmarking changes on a consistent multi-segment index even when they require re-indexing.
```

Or so?

Reply:
Sure, I'll add that, thanks!